src/components/Tabs/Tabs.tsx

```diff
@@ -12,15 +13,68 @@ interface ITabsProps {
 }
 
 export const Tabs = ({ children }: ITabsProps) => {
+  const [showScrollButton, setShowScrollButton] = useState(false);
+  const [disableScrollLeft, setDisableScrollLeft] = useState(true);
+  const [disableScrollRight, setDisableScrollRight] = useState(false);
+  const tabContent = useRef<HTMLUListElement>(null);
+
+  const onResize = useCallback(() => {
```

Comment:
Why not to put `onResize` to `useEffect` directly to avoid additional dependencies, `useCallback` usage, etc?